### PR TITLE
7400 governance docs

### LINKED ISSUE #7400: Document governance powers granted to the EC in MN 1B release

### PR DESCRIPTION
**Pull request by turadg**
closes: #7400

## Description

Refreshes documentation of  governance APIs and params in Inter Protocol.

Better would be to have the docs generated from code like https://github.com/Agoric/agoric-sdk/pull/7368 does but this is economical for now.

### Security Considerations

n/a
### Scaling Considerations

n/a

### Documentation Considerations

docs!

### Testing Considerations

nothing automated

### CODE REVIEW

**Comment by dckc** [comment]
any particular reason to rename the file?

**Comment by turadg** [comment]
> any particular reason to rename the file?

it's now about all governance and not just params. e.g. EC can call governed apis

File: packages/inter-protocol/docs/governance.md [hunk 1]
```diff
@@ -70,3 +96,18 @@ In `packages/vats/src/provisionPool.js`:
...
+([source](
+https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/auction/params.js#L58-L68)))
+
+| Governance Key            | Type          | WP? |
+| ------------------------- | :------------ | --- |
+| StartFrequency            | RelativeTime  | N/A |
+| ClockStep                 | RelativeTime  | N/A |
+| StartingRate              | BasisPoints   | N/A |
+| LowestRate                | BasisPoints   | N/A |
+| DiscountStep              | BasisPoints   | N/A |
```
Comment (dckc) [inline packages/inter-protocol/docs/governance.md:111]: If we're laying an interpretation NatValue here, should we do likewise for ChargingPeriod, RecordingPeriod above? Those are relative times in seconds, yes?

File: packages/inter-protocol/docs/governance.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+# Inter Protocol Governance
+
+Inter Protocol is subject to two forms of governance. Like the entire chain, it's subject to BLDer DAO governance. This is sometimes called the "big hammer" because the DAO can approve running any code in the core ("core eval").
+
+There is also governance by a contract governor contract (see `@agoric/governance`). This allows invited members to execute governed API methods or set governed parameters. In the conventional configuration of the Inter Protocol contracts there is one governing group, which is called the "Economic Committee".
```
Comment (dckc) [inline packages/inter-protocol/docs/governance.md]:
```suggestion
There is also governance by a contract governor contract (see `@agoric/governance`). This allows elected members to execute governed API methods or set governed parameters. In the conventional configuration of the Inter Protocol contracts there is one governing group, which is called the "Economic Committee".
```

or just strike "invited"... or go with "committee members"

Comment (dckc) [reply packages/inter-protocol/docs/governance.md]: wait... this is before we introduce the EC. "This allows a configurable electorate to execute..." would be more precise, though I'm not sure it's much more clear. hm.

"This delegates executing ... and setting ... to a body chosen when the contract is started"?

**Review by dckc — APPROVED** [review]
Looks good overall.

"invited" would be critical to fix if I could figure out a clearly better alternative.

I think the cost of renaming the file (breaking incoming links) outweighs the benefit of better matching its scope. But I suppose we've reached stalemate on that discussion, and I can live with this.

File: packages/inter-protocol/docs/governance.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+- [psm](https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/proposals/startPSM.js#L189-L193)
+- [reserve](https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/proposals/committee-proposal.js#L89-L94)
+- [smartWallet provisionPool](https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/vats/src/core/startWalletFactory.js#L248-L253)
+- [vault factory](https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/proposals/committee-proposal.js#L95-L97)
+
+
+## API Methods
+
+The Zoe contract framework to allow contracts to set filters on what offers they accept, by matching a string in the offer description. The governed contracts above grant to EC the ability to `setOfferFilter` that match a certain string description. (Note this has to be an exact match or a prefix match. [Other criteria are not yet supported](https://github.com/Agoric/agoric-sdk/issues/7317).)
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/docs/governance.md]:
```suggestion
The Zoe contract framework allows contracts to set filters on what offers they accept, by matching a string in the offer description. The governed contracts above grant to EC the ability to call `setOfferFilter()` to set a list of strings. (Offers are blocked when their 'descriptions are an exact match or a prefix match to one of the strings in the list. [Other criteria are not yet supported](https://github.com/Agoric/agoric-sdk/issues/7317).)
```

File: packages/inter-protocol/docs/governance.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+Specific contracts also grant other API methods to the EC:
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/docs/governance.md]:
```suggestion
Specific contracts also grant to the EC permission to invoke other API methods:
```

File: packages/inter-protocol/docs/governance.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+the type of each parameter, and an indicator of whether that parameter is described in
+the Inter Protocol Whitepaper, v0.8.  
+
+### Vault Factory - Director
+([source](https://github.com/Agoric/agoric-sdk/blob/5cbc847618dfb22f80e4641e5221b80e6daa109a/packages/inter-protocol/src/vaultFactory/params.js#L35-L39))
+
+| Governance Key     | Type              | WP? |
+| ------------------ | :---------------- | --- |
+| MinInitialDebt     | Amount            |     |
+| EndorsedUi         | Ratio             | No  |
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/docs/governance.md]:
```suggestion
| EndorsedUi         | String             | No  |
```

I'm not sure it's a string, but it's definitely not a Ratio.

File: packages/inter-protocol/docs/governance.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+### stakeFactory (unreleased)
+
+In `packages/inter-protocol/src/stakeFactory/stakeFactory.js`:
+
+| Governance Key     | Type                | WP? |
+| ------------------ | :------------------ | --- |
+| DebtLimit          | ParamTypes.AMOUNT   | Yes |
+| InterestRate       | ParamTypes.RATIO    | Yes |
+| LoanFee            | ParamTypes.RATIO    | Yes |
+| MintingRatio       | ParamTypes.RATIO    | Yes |
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/docs/governance.md]: for consistency
```suggestion
| DebtLimit          | Amount   | Yes |
| InterestRate       | Ratio   | Yes |
| LoanFee            | Ratio    | Yes |
| MintingRatio       | Ratio    | Yes |
```

**Review by Chris-Hibbert — COMMENTED** [review]
Just getting started, but I ran out of time. I'll take another look next week.

File: packages/inter-protocol/docs/governance.md (hunk 1 above)
Comment (turadg) [reply packages/inter-protocol/docs/governance.md:111]: good catch, thanks

File: packages/inter-protocol/docs/governance.md (hunk 4 above)
Comment (turadg) [reply packages/inter-protocol/docs/governance.md]: done

File: packages/inter-protocol/docs/governance.md (hunk 3 above)
Comment (turadg) [reply packages/inter-protocol/docs/governance.md]: done

File: packages/inter-protocol/docs/governance.md (hunk 2 above)
Comment (turadg) [reply packages/inter-protocol/docs/governance.md]: good improvements.

**Pull request merged**